hoomd/mpcd/ConfinedStreamingMethod.h

```diff
@@ -108,7 +109,7 @@ void ConfinedStreamingMethod<Geometry>::stream(unsigned int timestep)
     ArrayHandle<Scalar4> h_pos(m_mpcd_pdata->getPositions(), access_location::host, access_mode::readwrite);
     ArrayHandle<Scalar4> h_vel(m_mpcd_pdata->getVelocities(), access_location::host, access_mode::readwrite);
     const Scalar mass = m_mpcd_pdata->getMass();
-
+    ArrayHandle<unsigned char> mask_collision(m_should_mask_collide, access_location::host, access_mode::readwrite);
```

Comment:
You are going to overwrite whatever is here:
```suggestion
    ArrayHandle<unsigned char> h_bounced(m_bounced, access_location::host, access_mode::overwrite);
```
By convention, HOOMD replaces `m_` with `h_` for host handles and `d_` for device handles.